build.gradle.kts

```diff
@@ -95,6 +94,8 @@ tasks {
             properties("buf.license.header.holder").get(),
             "--year-range",
             properties("buf.license.header.range").get(),
+            "--ignore",
+            "/cachev",
```

Comment:
This is already part of the task below - just forgot to add to this one. We don't want to add the Buf license information to third party .proto sources we're using for testing.